src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table.helpers.ts

```diff
@@ -159,7 +159,7 @@ export class DatasetFlowTableHelpers {
                     case "FlowStartConditionBatching":
                         return `deadline time: ${moment(node.startCondition.batchingDeadline).fromNow()}`;
                     default:
-                        throw new Error("Unknown flow start condition");
+                        return "Unknown flow start condition";
```

Comment:
I don't really want to see any text in this case.

Reply:
Fixed